com.sudovanilla.penpot-desktop.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: penpot-desktop.desktop
+rename-icon: penpot-desktop
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    post-install:
```

Comment:
this should be in build-commands since it's `buildsystem: simple`

Reply:
Seems fair enough.